### DISCUSSION TITLE
Xttsv2 Training script

### DISCUSSION THREAD

**Discussion by IIEleven11** [coqui-ai/TTS#3773] [General]
Is there a reason that the example script and the demo script pull the 2.0.1 model to use as a base for fine tuning Instead of 2.0.2? 
    Also the Multi step learning rate/solution:
` lr=5e-06, 
        lr_scheduler="MultiStepLR",
        # it was adjusted accordly for the new step scheme
        lr_scheduler_params={"milestones": [50000 * 18, 150000 * 18, 300000 * 18], "gamma": 0.5, "last_epoch": -1},`
        
    Maybe I am misunderstanding something but from what I can tell, no one will ever engage the multi step learning rate when such large numbers are used. I know your first thought will be that those numbers represent steps. That's what I thought too so I adjusted them to 30, 60, and 90. It turns out those numbers represent epochs, which makes the default numbers used even more strange. Can anyone shed any light on this please?
